### DISCUSSION TITLE
[3차 데모데이] 회고 및 앞으로 계획 세우기

### DISCUSSION THREAD

**Discussion by Minjoo522** [woowacourse-teams/2024-devel-up#268] [백로그 논의]
# 피드백

### 백, 프론트 페어 프로그래밍
- 어려운 문제를 백, 프론트 같이 페어 해보는 건 어떤지?
- 제품의 관점에서 페어하기
- 서로의 요구사항을 이해하는 과정

### 프론트엔드
- GA 지표, 에러 관련 추가해야 할 것들에 대한 계획
	- 상태 코드 분류 ➡️ 에러 등급 ➡️ 라벨 ➡️ 슬랙 알람
- 분류한 에러가 발생한 뒤 어떻게 할 것인지?
	- 빠르게 처리하겠다
		- 빠르게 처리하겠다는 게 어떻게 하겠다는 거죠?
	- 공원 : 디버깅, 문제가 생겼을 때 파악 후 수정해서 내보내겠다는 것이다. 에러가 생겼을 때 어떻게 확인하고 수정해서 어떻게 내보낼지도 같이 생각을 해야 한다.
	- 구구 : 모든 에러를 대응해야하나도 고민해야 한다.
	- 네오 : 정책은 사람마다 생각하는 게 다르기 때문에 어떻게 대처할지 규칙을 정해야 한다(돌아가면서 한다거나 담당자가 한다거나)

### 백엔드
- 에러 로그 타입으로 보기만 해도 될듯
- 하는김에 했다는 말은 하지 않았으면 좋겠다
- Nginx는 로그 어떻게 남길건지?
- 왜 하려고 했는지?
- 우리 서비스에서 어떤 부분을 모니터링할지로 시작된게 아니라 결과와 중간 과정에 어떤 의사결정이 있었는지가 비어져있다. 왜 이렇게 발전을 했는지 비어있다. 요구 사항을 받고 어떤 선택지들이 있었고, 어떤 논의가 있어서 이렇게 결정하게 됐는지?
	- 결과적으로 프로덕션으로 나갔을 때 모니터링조차 관리해야하는 건데 어떻게 빠르게 적용하고 변화할 수 있는지도 고려 대상이다. 히스토리가 있는지?
	- 러닝 커브 : 학습의 비용으로 설득이 잘 안된다.
		- 우리 서비스가 모니터링이 필요해?부터 시작해서 어떤 의사결정과 충돌이 있었는지 궁금하다.
		- 어떤 좌충우돌이 있었는지 보고싶었다.
- 기술 선택의 이유가 말이 안된다?
- 의사 결정 과정이나 고민한 기록

### 기타
- 그럴듯한 컨텐츠 추가하기

**Comment by chosim-dvlpr** [comment]
정리 감사합니당 👍🚀
